recipes/pytest-astropy-header/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytest-astropy-header" %}
+{% set version = "0.1.1" %}
+{% set git_url = "https://github.com/astropy/pytest-astropy-header" %}
+{% set sha256 = "928547645e53586ff57c9e00ed050b876d4dd53dce95f7e6b0130851a4f71aec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest >=3.0
+  run:
+    - python
+    - pytest >=3.0
+
+test:
+  imports:
+    - pytest_astropy_header
+    - pytest_astropy_header.display
+
+about:
+  home: {{ git_url }}
+  license_family: BSD
+  license: BSD 3-Clause
+  license_file: LICENSE.rst
+  summary: 'Pytest plugin to add diagnostic information to the header of the test output'
+  doc_url: {{ git_url }}
+  dev_url: {{ git_url }}
+
+extra:
+  recipe-maintainers:
+    - astrofrog-conda-forge
+    - bsipocz
```

Comment:
@bsipocz Are you OK with being a maintainer?

Reply:
yes, I'm, in any feedstocks coming from astropy/astrofrog/etc.